app/controllers/menus_controller.rb

```diff
@@ -1,4 +1,10 @@
 class MenusController < ApplicationController
-  def index    
+  skip_before_action :authenticate_employee!, only: [:home]
+
+  def index
+  end
+
+  def home
+    @companies = Company.all
```

Comment:
where is this used?
this will give all companies

Reply:
will take you to the home page where employee will select which company he/she belongs to and then sign in

Reply:
Company.all will give "all" companies, not necessarily current_user's companies. No?

Reply:
yes and I need all companies as well in the root page of site